kolibri/plugins/coach/assets/src/modules/pluginModule.js

```diff
@@ -21,7 +21,7 @@ const logging = logger.getLogger(__filename);
 export default {
```

Comment:
**non-blocking but potentially important to address in a further loading state or coach-specific PR if not here:**

i just wanted to re-raise the issue here as it is not caused or worsened by your work but is slightly affected by it: the oddness in loading in `Coach` as a multi-facility user (outlined in #10714 & detailed further [in this comment](https://github.com/learningequality/kolibri/issues/10714#issuecomment-1558262576)) was also noticeable during review of your PR, because now when a multi-facility user goes to look at the classes in one of their imported facilities and the default facility's class list is briefly displayed before the imported facility's class list renders and replaces it, the default facility's class list is also decorated with a loading spinner. 

https://github.com/learningequality/kolibri/assets/43046460/0efde4e2-d1fe-42ce-b21c-04656334f3eb

this definitely was not caused by your PR & just FYI the same doesn't occur in the somewhat parallel scenario in `Facility` - it's all looking good there:

https://github.com/learningequality/kolibri/assets/43046460/5a33235e-5a24-439c-8e54-6132c9b0d8d5

Reply:
Hmm... I'll see if maybe we can clear the contents of the page in coach whenever we make a new requests to a different facility or something.